### ISSUE TITLE
detect multiple objects using same bounding box

### ISSUE THREAD

**Issue by tomriddle54** [AlexeyAB/darknet#6436]
Hi @AlexeyAB is it possible to detect multiple objects using same bonding box example I have bike photos with person sitting on it if I label bike as 5x5 box and I keep Same box to person. Will it tell bike and person on prediction?

**Comment by fareed05** [comment]
No, it is not possible to detect multiple objects with the same bounding box.. you should create a separate bounding box for the person. 
Thanks
